### PR TITLE
[QC-1097] - use metadata to filter what to delete 

### PR DESCRIPTION
**Pull request by Barthelemy**
with o2-qc-repo-delete-time-interval

### CODE REVIEW

File: Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-delete-time-interval [hunk 1]
```diff
@@ -6,23 +6,29 @@ import dryable
...
-    parser.add_argument('--only-path-no-subdir', action='store_true', help='Do not process the subfolders, i.e. do not '
-                                                                 'add .* at the end of the path.')
+    parser.add_argument('--only-path-no-subdir', action='store_true',
+                        help='Do not process the subfolders, i.e. do not add .* at the end of the path.')
     parser.add_argument('--one-by-one', action='store_true', help='Ask confirmation for each deletion')
-    parser.add_argument('--print-list', action='store_true', help='Only print the list of objects that would be deleted')
+    parser.add_argument('--print-list', action='store_true',
+                        help='Only print the list of objects that would be deleted')
+    parser.add_argument('--metadata', dest='metadata', action='store', default="",
+                        help='Delete only versions matching these metadata')
```
Comment (knopers8) [inline Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-delete-time-interval:31]: can you explain the expected format in the argument help?

**Review by knopers8 — APPROVED** [review]

File: Framework/script/RepoCleaner/README.md [hunk 2]
```diff
@@ -72,5 +72,14 @@ Prerequisite
...
+3. `python3 -m twine upload --repository pypi dist/*`
+
+## Use venv
+
+1. cd Framework/script/RepoCleaner
+2. python3 -m venv env
+3. source env/bin/activate
+4. python -m pip install -r requirements.txt
+5. python3 -m pip install . 
+6. You can execute and work. Next time just do "activate" and then you are good to go
```
[the remark this answers is not in the document]
Comment (Barthelemy) [reply Framework/script/RepoCleaner/README.md:85]: You don't need to use it but it is a way to create a "virtual environment", i.e. a kind of "container" with its own set of interpreter and libraries on top of the ones from the system. It is useful if you develop things in parallel and need different environment for different projects. 

https://docs.python.org/3/library/venv.html#

[the remark this answers is not in the document]
Comment (knopers8) [reply Framework/script/RepoCleaner/README.md:85]: thanks

File: Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-delete-time-interval (hunk 1 above)
Comment (Barthelemy) [reply Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-delete-time-interval:31]: yes, I have put it somewhere else but it would be fitting here.

**Pull request merged**